kyo-prelude/shared/src/main/scala/kyo/kernel/Pending.scala

```diff
@@ -84,11 +66,53 @@ object `<`:
             mapLoop(v)
         end map
 
-        inline def evalNow(using inline flat: Flat[A]): Maybe[A] =
```

Comment:
Moved to the end + made `private[kyo]`. I don't envision users needing this API and it exposes too much of Kyo's current internal implementation, which might change over time.